### PR TITLE
New: Carousel for marketplace landing

### PR DESCRIPTION
**Pull request by patrick-ivers**
#### Changes
- Add Carousel component for the marketplace landing page

#### Screenshots
carousel

### CODE REVIEW

File: src/components/adslotUi/CarouselComponent.jsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  return (
+    <Carousel decorators={decorators} {...props} className={classNames(baseClass, className)}>
+      {children}
+    </Carousel>
+  );
+};
+
+CarouselComponent.propTypes = {
+  className: PropTypes.string,
+  children: PropTypes.node,
```
Comment (omgaz) [inline src/components/adslotUi/CarouselComponent.jsx]: Should this be required?

File: package.json [hunk 2]
```diff
@@ -76,13 +76,14 @@
     "mocha": "^3.2.0",
     "moment": "^2.17.1",
     "node-sass": "^3.13.0",
+    "nuka-carousel": "^2.0.4",
```
Comment (lteacher) [inline package.json]: `nuka-cola`?
image

File: src/components/adslotUi/CarouselComponent.jsx (hunk 1 above)
Comment (patrick-ivers) [inline src/components/adslotUi/CarouselComponent.jsx]: I didn't want to make it required as slides might need to lazy load, so for a brief moment they may not be present in the component.

**Comment by patrick-ivers** [comment]
[(3) Marketplace landing: component - carousel](https://trello.com/c/sDm8wZFA/6317-3-marketplace-landing-component-carousel)

File: src/components/Main.jsx [hunk 3]
```diff
@@ -655,6 +656,27 @@ class AppComponent extends React.Component {
...
+          <div style={{ position: 'relative' }}>
+            <h1
+              style={{
+                position: 'absolute',
+                right: 0,
+                left: 0,
+                'text-align': 'center',
+                color: '#fff',
+              }}
+            >ＡＥＳＴＨＥＴＩＣＳ</h1>
```
Comment (omgaz) [inline src/components/Main.jsx]: Aesthetic

**Comment by patrick-ivers** [comment]
[(2) Marketplace landing: interface - collection carousel](https://trello.com/c/D7FojRZB/6353-2-marketplace-landing-interface-collection-carousel)

**Pull request merged**